src/server/graphql/models/Team/archiveTeam/archiveTeam.js

```diff
@@ -0,0 +1,58 @@
+import getRethink from 'server/database/rethinkDriver';
+import {
+  requireSUOrTeamMember,
+  requireWebsocket
+} from 'server/utils/authorization';
+import {handleSchemaErrors} from 'server/utils/utils';
+import {
+  GraphQLNonNull,
+  GraphQLBoolean,
+} from 'graphql';
+import {TeamInput} from '../teamSchema';
+import shortid from 'shortid';
+import {TEAM_ARCHIVED} from 'universal/utils/constants';
+import archiveTeamValidation from './archiveTeamValidation';
+
+export default {
+  type: GraphQLBoolean,
+  args: {
+    updatedTeam: {
+      type: new GraphQLNonNull(TeamInput),
+      description: 'The input object containing the teamId and any modified fields'
+    }
+  },
+  async resolve(source, {updatedTeam}, {authToken, socket}) {
+    const r = getRethink();
+
+    // AUTH
+    requireSUOrTeamMember(authToken, updatedTeam.id);
+    requireWebsocket(socket);
+
+    // VALIDATION
+    const {errors, data: {id, name, isArchived}} = archiveTeamValidation()(updatedTeam);
+    handleSchemaErrors(errors);
+
+    // RESOLUTION
+    await r.table('Team').get(id).update({isArchived});
+
+    setTimeout(async () => {
```

Comment:
Here are all **4** queries wrapped up in a ReQL query for you. Take note, you'll have to bring your own variables :)

```javascript
r.db('actionDevelopment').table('Team').get('Sk6bgx_2e')
  .pluck('orgId')
  .do((doc) => {
    return ({
      orgId: doc('orgId'),
      userIds: r.db('actionDevelopment').table('TeamMember').getAll('Sk6bgx_2e', {index: 'teamId'})
                 .pluck('userId')
                 .coerceTo('array')
                 .map((doc) => doc('userId'))
    })
  })
  .do((doc) => {
    return r.db('actionDevelopment').table('Notification').insert({
      id: 'CHANGEME_shortid',
      type: 'TEAM_ARCHIVED',
      startAt: r.now(),
      orgId: doc('orgId'),
      userIds: doc('userIds'),
      varList: ['aName']
    })
  })
  .do(() => {
    return r.db('actionDevelopment').table('Team').get('Sk6bgx_2e').update({isArchived: true})
  })    
```